lithium/lithium.py

```diff
@@ -55,7 +57,8 @@ def readTestcase(self, filename):
             # Determine whether the f has a DDBEGIN..DDEND section.
             for line in f:
                 if line.find(b"DDEND") != -1:
-                    raise LithiumError("The testcase (%s) has a line containing 'DDEND' without a line containing 'DDBEGIN' before it." % self.filename)
+                    raise LithiumError("The testcase (%s) has a line containing 'DDEND' " % self.filename +
+                                       "without a line containing 'DDBEGIN' before it.")
```

Comment:
This could be

    ("...%s..."
     "... ... " % var)

or does pylint/flake8 complain about that? No biggie either way.